tests/integration/utils.go

```diff
@@ -222,6 +223,9 @@ func ExecuteTestCase(
 	databases = skipIfDatabaseTypeUnsupported(t, databases, testCase.SupportedDatabaseTypes)
 	clients = skipIfClientTypeUnsupported(t, clients, testCase.SupportedClientTypes)
 
+	// We do this because for now JSClient only supports badger in-memory database.
+	clients = skipJSClientIfUnsupportedDBType(t, clients, databases)
```

Comment:
suggestion: This seems overly restrictive. I would suggest doing the check for the specific database bellow instead of completely removing the client if a single database is incompatible.

Reply:
If client is JS type then we only want to use compatible DB types (there is only one DB type that JS supports - `Badger In Memory`). So while it looks like we are removing the entire client, it is because it doesn't support any other database types if the compatible DB type is not found.

Keeping as is.

Reply:
If we have clients[JS] and database[BadgerInMemory, BadgerFile]. It will skip all of it instead of just skipping the BadgerFile database. It seems wrong and can cause devs to think the test is passing while in fact it's skipped.

Reply:
> If we have clients[JS] and database[BadgerInMemory, BadgerFile]. It will skip all of it instead of just skipping the BadgerFile database. It seems wrong and can cause devs to think the test is passing while in fact it's skipped.

Thanks for flagging, I forgot why I did this now. Will investigate.

Reply:
Removed the JS skipping logic, whatever was leaking and running JS tests to run and overwrite the db type seems to have been fixed in upstream.

Added two more commits:

- [x] Small bug in testing framework output: ``` PR(FIX): Invalid client type showing db type instead ```
- [x] Remove the JS skipping: ``` PR(REVIEW): Remove the JS skipping, no longer needed ```